README.md

```diff
@@ -35,6 +35,27 @@ You can use the following command if you want to deploy `kafka` and related serv
 $ docker-compose -f docker-compose.yml -f docker-compose.kafka.yml up -d 
 ```
 
+Setup pgadmin to view moodle database
+
+To log into the server go to [http://localhost:5050/](http://localhost:5050/)
+
+Log in:
+
+- Email: admin@admin.com
+- Password: root
+
+On the home page click "Add new server"
+Give the new server a name. Example: "my_db"
+
+Go to the connection tab and set the following perameters
+
+- Hostname/address: postgres_db
```

Comment:
We can just use `postgres` directly I believe (see related comment in the `docker-compose.yml`).